content/en/style-guide/_index.md

```diff
@@ -25,19 +25,19 @@ Additionally, it follows the following rules:
 
 ## Audience
 
-The Glossary is written for a technical AND non-technical audience. 
-Please ensure definitions are explained in simple terms and don’t assume technical knowledge. More do that below under Definition. 
+The Glossary is written for technical *and* non-technical audiences.
+Please explain definitions in simple terms, and don’t assume technical knowledge. More about this is below in [Definition](#definition).
 
-## Minimal Viable Definition 
+## Minimum viable definition 
 
-Our goal is to make it really easy for anyone to understand cloud native terms. 
+We aim to make it easy for anyone to understand cloud native terms. 
 As such, we focus on simplicity. 
-That means using clear and simple language with examples anyone who uses technology can relate to (more to that below) but also providing a *minimal viable definition*, at least from a technical point of view. 
```

Comment:
Yeah, the idea for the term comes from minimal viable product. Since most people are familiar with it, I thought it might stick better than explaining it with different words.